banking/klarna_kosma_integration/doctype/banking_settings/banking_settings.py

```diff
@@ -86,26 +87,63 @@ def daily_sync_ebics():
 			_("Developer mode is enabled. Please disable it to continue auto-syncing bank transactions.")
 		)
 
-	yesterday = (now_datetime() - timedelta(days=1)).date().isoformat()
-	for ebics_user in frappe.get_all(
+	yesterday = (now_datetime() - timedelta(days=1)).date()
+	for ebics_user, country in frappe.get_all(
 		"EBICS User",
 		filters={
 			"initialized": 1,
 			"bank_keys_activated": 1,
 			"passphrase": ("is", "set"),
 			"keyring": ("is", "set"),
 		},
-		pluck="name",
+		fields=["name", "country"],
+		as_list=True,
 	):
+		if successful_request_exists(
+			ebics_user, order_type="Z53" if country == "Switzerland" else "C53", request_date=yesterday
+		):
+			continue
+
 		frappe.enqueue(
 			sync_ebics_transactions,
 			requested_by="System",
-			start_date=yesterday,
-			end_date=yesterday,
+			start_date=yesterday.isoformat(),
+			end_date=yesterday.isoformat(),
 			ebics_user=ebics_user,
 		)
 
 
+def successful_request_exists(ebics_user: str | None, order_type: str, request_date: date) -> bool:
+	existing_request = frappe.db.get_value(
+		"EBICS Request",
+		{
+			"ebics_user": ebics_user,
+			"order_type": order_type,
+			"status": "Successful",
```

Comment:
If we want to be a little more performant, we could narrow the results:
```suggestion
			"ebics_user": ebics_user,
			"order_type": order_type,
			"requested_by": "System",
			"status": "Successful",
```

Reply:
When a user request was successful, we don't need to do another system request.